docs/sources/alerting/set-up/configure-roles/index.md

```diff
@@ -0,0 +1,79 @@
+| Admin  | Write access to alert rules, notification resources (notification API, contact points, templates, time intervals, notification policies, and silences), and provisioning. |
+| Editor | Write access to alert rules, notification resources (notification API, contact points, templates, time intervals, notification policies, and silences), and provisioning. |
+| Viewer | Read access to alert rules, notification resources (notification API, contact points, templates, time intervals, notification policies, and silences).                    |
+
+## Assign roles
+
+To assign roles, admins need to complete the following steps.
+
+1. Navigate to **Administration** -> **Users and access** -> **Users, Teams, or Service Accounts**.
+1. Search for the user, team or service account you want to add a role for.
+1. Add the role you want to assign.
+
+## Manage access using folder permissions
+
+You can further customize access for alert rules, simplified alert routing, and provisioning by assigning permissions to individual folders.
+
+This prevents every user from having access to modify all alert rules and gives them access to the folders with the alert rules they are working on.
+
+For example, if you are using simplified alert routing and adding contact points to your alert rules, it also helps you avoid the scenario where someone from another team accidentally removes the wrong notification policy or adds a competing one, and all of a sudden you stop getting your notifications.
+
+In this case, you would assign the **Viewer** role and then add **Editor** permission to the folder. Adding the **Editor** permission to the folder does not overwrite the **Viewer** role.
+
+Details on the adding folder permissions as well as roles and the access that provides for Grafana Alerting is below.
+
+| Role   | Folder permission | Access                                                                                    |
+| ------ | ----------------- | ----------------------------------------------------------------------------------------- |
+| Admin  | -                 | Write access to alert rules in all folders.                                               |
+| Editor | -                 | Write access to alert rules in all folders.                                               |
+| Viewer | Admin             | Read access to alert rules in all folders. Write access to alert rules **only** in the folders where the Admin permission is added.  |
+| Viewer | Editor            | Read access to alert rules in all folders. Write access to alert rules **only** in the folders where the Editor permission is added. |
+| Viewer | Viewer            | Read access to alert rules in all folders.                                                               |
+
+{{% admonition type="note" %}}
+You cannot use folders to customize access to notification resources.
+{{% /admonition %}}
```

Comment:
Same behavior but `{{<` is consistent with Writers' Toolkit (https://grafana.com/docs/writers-toolkit/write/shortcodes/#admonition)

nit: tone (https://grafana.com/docs/writers-toolkit/write/style-guide/voice-tone-guidelines/#tone)

```suggestion
{{< admonition type="note" >}}
You can't use folders to customize access to notification resources.
{{< /admonition >}}
```